app/api/v1/routers/build_router.py

```diff
@@ -0,0 +1,83 @@
+from fastapi import Depends, HTTPException, APIRouter
+from sqlalchemy.orm import Session
+from uuid import UUID
+
+from app.db.database import get_db
+from app.db import crud
+# from app.schemas.build import _
+# To do: Set up schemas for builder
+
+
+router = APIRouter()
+
+
+@router.get("/products/keycaps")
+def get_keycaps(db: Session = Depends(get_db)):
```

Comment:
Return type annotations for functions in this file.

Return to parts.

Same for all